### PR TITLE
Review Reporting Form for design inconsistencies

### PR DESCRIPTION
**Pull request by sharkcrayon**
[Clubhouse Story](https://app.clubhouse.io/augur/story/7289)

## Submitter checklist
- [x] Test covering functionality added/fixed in
- Check the linked story includes the following:
  - [x] Steps to reproduce
  - [x] Screenshot (If applicable)

## Reviewer Checklist
- [ ] Test/lint pass 
- [ ] Post merge, story marked complete

### CODE REVIEW

**Comment by justinbarry** [comment]
@sharkcrayon Can you resolve the conflict here? Please and thank you.

File: src/modules/common/components/chevron-flip/chevron-flip.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const ChevronFlip = p => (
+  <svg
+    className={classNames(
+      Styles.ChevronFlip,
+      p.className,
+      { [Styles.pointDown]: p.pointDown },
+    )}
+    viewBox="0 0 16 16"
+  >
+    <g id="Symbols" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd" strokeLinecap="round" strokeLinejoin="round">
```
Comment (justinbarry) [inline src/modules/common/components/chevron-flip/chevron-flip.jsx]: Are the IDs necessary? If so, can they be classes?

File: src/modules/reporting/components/reporting-report/reporting-report.jsx [hunk 2]
```diff
@@ -123,6 +123,7 @@ export default class ReportingReport extends Component {
             location={p.location}
             history={p.history}
             cardStyle="single-card"
+            hideReportEndingIndicator
```
Comment (justinbarry) [inline src/modules/reporting/components/reporting-report/reporting-report.jsx]: I can't find this being set anywhere. Where should it be set?

File: src/modules/common/components/chevron-flip/chevron-flip.jsx (hunk 1 above)
Comment (sharkcrayon) [reply src/modules/common/components/chevron-flip/chevron-flip.jsx]: No, the ids are not necessary. It's just how the SVG is auto-exported from Sketch. I've removed them.

File: src/modules/reporting/components/reporting-report/reporting-report.jsx (hunk 2 above)
Comment (sharkcrayon) [reply src/modules/reporting/components/reporting-report/reporting-report.jsx]: It's not set anywhere. Passing it in like this is treated as `hideReportEndingIndictor={true}`. I thought it was common practice for us to do it this way? If not, I can be more verbose.

Comment (justinbarry) [reply src/modules/reporting/components/reporting-report/reporting-report.jsx]: Really? I did not know that! Cool trick! I figured it came through as undefined.
If that is the case I'm totally happy with it.

**Pull request merged**